src/extensions/default/bramble/lib/ConsoleManager.js

```diff
@@ -15,8 +17,30 @@ define(function (require, exports, module) {
         var args = data.args;
         var type = data.type || "log";
 
-        // TODO: Show this in Custom Console UI, see issue #1675 in Thimble
+        if (type === "error-handler") {
+            var regex = new RegExp('(blob:.+):([^:]+):(.+)', 'gm'),
+                endingRegex = new RegExp(':([0-9]+):([0-9]+)$', 'gm'),
+                stackTrace = args["stack"].split("@"),
+                newArgs = [];
+            
+            // Handle Blob URLs
+            for(var i = 1; i < stackTrace.length; i++){
+                var stackItem = stackTrace[i].match(regex);
+                stackItem = (BlobUtils.getFilename(stackItem[0].replace(endingRegex, ''))).split('/');
```

Comment:
` "testerConsole@blob:http://localhost:8000/4eaede7e-49a6-4c1e-997f-3da508f77cfa:2100:9onclick@blob:http://localhost:8000/4eaede7e-49a6-4c1e-997f-3da508f77cfa:1:1"`

Splitting to isolate the blob URL from the function call. 

By calling `Path.basename(...)`, what does the function variable return?